src/utils/useNavigation.test.js

```
@@ -334,15 +318,19 @@ describe('useNavigation', () => {
+          id: 'insights',
+          title: 'RHEL',
+          navItems: [navItem],
+        },
+      ]);
       const wrapper = ({ children }) => (
-        <RouterDummy initialValues={[[navigationAtom, navigation]]} store={store} path="/insights/dashboard">
+        <RouterDummy
+          initialValues={[[navigationAtom, navigation]]}
+          store={store}
+          path="/insights/dashboard"
+        >
```

ESLint

[error] Replace `⏎··········initialValues={[[navigationAtom,·navigation]]}⏎··········store={store}⏎··········path="/insights/dashboard"⏎········` with `·initialValues={[[navigationAtom,·navigation]]}·store={store}·path="/insights/dashboard"`

(prettier/prettier)